pkg/docker/docker_test.go

```diff
@@ -11,12 +10,8 @@ import (
 var testDir = "../../testdata/intoto"
 
 func TestRun(t *testing.T) {
-	image := "radumatei/in-toto-container:v1"
-	layout := filepath.Join(testDir, "demo.layout.template")
-	key := filepath.Join(testDir, "alice.pub")
-	links := testDir
-	targets := []string{filepath.Join(testDir, "foo.tar.gz")}
-
-	err := Run(image, layout, key, links, log.InfoLevel.String(), targets)
+	// NOTE: Tag will be empty since we cannot inject build-time variables during testing.
+	// Therefore, we shall use the "latest" tag.
```

Comment:
I haven't tested this yet, but I would assume that the `Tag` variable (exported from the `docker` package) should be accessible here - it is empty because it is not currently set in the Makefile at all.